### PR TITLE
Create resolution-002.md

### PR DESCRIPTION
**Pull request by benhylau**

### CODE REVIEW

File: resolutions/resolution-002.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+2. Two new directors have been elected to the board.
+
+**IT WAS RESOLVED THAT:**
+
+1. The directors of the board unanimously put forward the bylaws for adoption by the membership.
+
+2. The directors of the board unanimously adopt an amendment of the articles for adoption.
+
+3. [  ] of the Co-operative is authorized to sign all documents and perform such acts as may be necessary or desirable to give effect to the above resolution.
```
Comment (benhylau) [inline resolutions/resolution-002.md:20]: @dcwalk what is `[  ]` supposed to be?

File: resolutions/resolution-002.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
+# ANNUAL RESOLUTIONS OF THE BOARD OF DIRECTORS
+
+<sup>[From template][template]</sup>
+
+**Resolution(s) in writing** of the board of directors of Hypha Worker Co-operative Inc. (the "Co-operative") 
+from the meeting of the directors held by electronic means on this 13 day of June, 2020.
```
Comment (benhylau) [inline resolutions/resolution-002.md:6]:
```suggestion
from the meeting of the directors held by electronic means on this 13th day of June, 2020.
```

File: resolutions/resolution-002.md [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+3. [  ] of the Co-operative is authorized to sign all documents and perform such acts as may be necessary or desirable to give effect to the above resolution.
+
+4. The following individuals are appointed and confirmed as officers of the Co-operative for a term of one year or until replaced:
+
+    - Yurko Ꭻаrеⅿ𝗄ס: President;
+    - Patrick Connolly: Vice-President;
+    - Udit Vira: Treasurer; and
+    - Benedict Lau: Secretary
+
+Dated in the Province of Ontario on the 13 day of June, 2020.
```
Comment (benhylau) [inline resolutions/resolution-002.md:29]:
```suggestion
Dated in the Province of Ontario on the 13th day of June, 2020.
```

File: resolutions/resolution-002.md (hunk 1 above)
Comment (dcwalk) [inline resolutions/resolution-002.md:20]:
```suggestion
3. Any two directors or officers of the Co-operative are authorized to sign all documents and perform such acts as may be necessary or desirable to give effect to the above resolution.
```

**Pull request merged**